controlled-vocabulary/src/main/java/org/uniprot/core/cv/xdb/UniProtXDbTypes.java

```diff
@@ -105,14 +105,12 @@ private void init() {
                                         uriLink,
                                         attributes,
                                         linkTp,
-                                        linkedDb);
+                                        linkedReason);
                         types.add(xdbType);
                     });
             typeMap =
                     types.stream()
-                            .collect(
-                                    Collectors.toMap(
-                                            UniProtXDbTypeDetail::getDisplayName, val -> val));
+                            .collect(Collectors.toMap(UniProtXDbTypeDetail::getName, val -> val));
```

Comment:
Could use Function.identify() instead of val->val if you think it's clearer. You choose, either is fine.